Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM fedora:latest
```

Comment:
Why not from `registry.access.redhat.com/ubi7/ubi:latest`?

Reply:
I took the docker image example from buildah repo. tbh I don't know what's the difference here, can you explain it?

Reply:
why not `ubi-minimal`?

Reply:
It could very well be ubi-minimal, depending on the needs of the `kfn` but I imagine that would be fine.

Reply:
Both ubi and ubi-minimal doesn't work due missing packages. I prefer to leave fedora for now, we'll improve it in the future

Reply:
Since we need this image now only to test the integration with the operator.